### ISSUE TITLE
When doing multi-comic scraping, an option should be present to automatically fetch the next set of volumes.

### ISSUE THREAD

**Issue by mcpierce** [comixed/comixed#25]
When scraping multiple comics, it would make it easier if there were a way to tell CX to automatically fetch the volumes for the next comic.

**Comment by bareheiny** [comment]
Can I just check that at the moment, each comic needs to have the "fetch" button pushed when scraping multiples?

When scraping multiple comics, I'd ideally like CX to automagically move onto the next issue to be scrapped...as well as automagically pulling the detials for a comic in a series that has already been scrapped (ie. if I've scrapped Liberty Meadows no1 sucessfully, no2 should be scrapped without any input).

Possibly may require configuration settings to allow users to auto-match or manually match each issue - similar to the CR ComicVine scraper.

**Comment by mcpierce** [comment]
Correct. If there isn’t one already then feel free to open a new ticket to
have multi-comic scraping move on to the next comic after scraping.
-- 
Sent from Gmail Mobile

**Comment by plumber-craic** [comment]
for a long queue of comics from the same series, it would save a lot of time to select just once which metadata source ID to use.

**Comment by mcpierce** [comment]
@plumber-craic Did you mark the metadata source as preferred on the configuration page?

**Comment by plumber-craic** [comment]
@mcpierce I have ComicVineMetadataAdaptor set as my preferred metadata source. I am trying to do this:

1. Select Multi Comic Scraping
2. Click Scrape This Comic
3. Click the thumbnail of 3rd one in the list
4. Click the thumb icon
5. Click Yes

All the other books in the same directory also belong to the 3rd one from the comicvine list, but I need to complete steps 2-4 for each book. If there are 50+ files in a directory the process will take a very long time. To safely automate a multi comic scrape I guess two things are needed: issue numbers and a single matching series to use from comicvine. Then I could say for these selected comics, use a single comicvine series, and because the issue numbers are in place the process can complete without intervention. In cases there the issue number is not present or is outside the range of the scraped series the process can stop for manual intervention.

**Comment by mcpierce** [comment]
@plumber-craic Ah, okay, I understand what you're saying. That's a feature that was requested in #1702 by morpheus200 and definitely on the road map.
